src/templates/ski-finder.tsx

```diff
@@ -55,14 +55,14 @@ export const transformProps: TransformProps<TemplateRenderProps> = async (
 };
 
 export const getPath: GetPath<TemplateRenderProps> = ({ document }) => {
-  return document.slug ?? "index.html";
+  return document.slug ?? document.id;
```

Comment:
will we have guarantees that `document.id` is unique? Want to avoid a url conflict error. If that is guaranteed unique, then I'm good

Reply:
There can't be duplicate entity ids in the KG, right?  I think it's guaranteed to always be unique.